### PR TITLE
concurrent helmfile template

### PR DESCRIPTION
**Pull request by rochana-atapattu**
#900 
this starts templating helmfiles in the first helmfile concurrently

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -243,6 +244,8 @@ replace (
 	// override the go-scm from tekton
 	github.com/jenkins-x/go-scm => github.com/jenkins-x/go-scm v1.11.5
 
+	github.com/jenkins-x/jx-helpers => /Users/rochana/Code/Devops/repos/jx-helpers
```
Comment (ankitm123) [inline go.mod:247]: Ready for review? Then you want to remove this replace ...

Comment (rochanaatapattu) [reply go.mod:247]: noted

Comment (rochanaatapattu) [reply go.mod]: i updated the code and did some refactoring. please let me know if i can improve. This is my first pr for and OSS project.

Comment (rochana-atapattu) [reply go.mod]: i updated the code and did some refactoring. please let me know if i can improve.

**Review by rochana-atapattu — COMMENTED** [review]
ignore

**Comment by rochana-atapattu** [comment]
` helmfiles.GatherHelmfiles(o.Helmfile, o.Dir)` return a duplicated list of helmfiles. Is there a functionality I can use to filter the duplicated?

**Pull request closed without merging**